### DISCUSSION TITLE
Add possibility to create multiple flutter projects with serverpod create

### DISCUSSION THREAD

**Discussion by Lord-Platypus** [serverpod/serverpod#2748] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Let's say that I want to create 2 apps (or maybe even 3) for a restaurant, with the same backend: one for the customers and one for the owner and waitresses. 

**Describe the solution you'd like**
 They need to share the same models, so I need to create 2 flutter folders 

**Describe alternatives you've considered**
I could create 2 serverpod projects, but I would need to copy and paste the yaml files with the models and generate again all the classes

**Comment by SandPod** [comment, marked as answer]
Hey @EmanueleVinci 👋 

You can create as many flutter projects as you want with `flutter create` and then import the shared client.
The client is updated when generating the project, but the flutter projects are never touched.

When running `serverpod create` we create a single flutter project as a starting point and to demonstrate how you should depend on the client to communicate what the server.

**Reply by Lord-Platypus** [reply]
I had missed a few steps reading the documentation, thank you very much.

P.S. Serverpod is great, thank you
